test/parallel/test-fs-promises.js

```diff
@@ -96,6 +98,9 @@ function verifyStatObject(stat) {
     await chmod(dest, 0o666);
     await fchmod(handle, 0o666);
 
+    await chown(dest, process.getuid(), process.getgid());
```

Comment:
`process.getuid()` and `process.getgid()` are not defined on Windows.

Reply:
@cjihrig 
Thank you for your review.
Fixed not to call the `process.getuid()` and `process.getgid()` on Windows.